Commonalities/documentation/Working/CAMARA-AuthN-AuthZ-Concept.md

```diff
@@ -141,6 +148,97 @@ The authorization code is a temporary code that the client will exchange for an
+
+3GPP TS 23.682 [13] specifies the _Domain Identifier_ as a domain that is under the control of a Mobile Network Operator (MNO) and then indicates that the _Domain Identifier_ is used to identify where services provided by the operator network can be accessed. It is possible for an operator to differentiate services by providing different domain identifiers associated to the same subscription.
+
+The _Local Identifier_, also according to 3GPP TS 23.682 [13], is used to derive or obtain the SUPI. It is unique within the applicable domain and managed by the Mobile Network Operator.
+
+Although the concepts of the _Domain Identifier_ and _Local Identifier_ were initially created for Cellular IoT environment, these are generally applicable to any type of user subscription in the mobile network.
+
+### AF-specific UE external identifier <a name="afspecific"></a>
+
+AF-specific UE external identifier is just a GPSI in the format of an External Identifier, that happens to be define for a specific AF, with the regular format of an External Identifier:
+
+\&lt;localIdentifier\&gt;@\&lt;AF-specific-domainRealm\&gt;
+
+The AF-specific UE external identifier is associated to the SUPI in UDM/UDR. It is assumed that the MNO provisions the AF-specific UE external identifier in UDM/UDR.
+
+The AF-specified UE external identifier was introduced in 3GPP Rel-17 to solve these problems:
+
+- A UE may change its UE IP address when it is served by an Edge Computing site.
+- If the AF identifies the UE with its UE IP address, the AF may lose control when the UE IP address changes.
+- The AF-specific UE external identifier takes into consideration privacy of the MSISDN, so that it does not require the user to disclose its MSISDN.
+
+The AF keeps the AF-specific UE external identifier for the duration of the AF session. The AF must not keep a binding to the UE IP address, since the UE IP address may change.
+
+3GPP indicates that the AF specific UE external identifier must not contain an MSISDN, due to privacy reasons. Please note that a GPSI can still contain an MSISDN, should this privacy reasons be properly addressed (e.g., the user discloses its MSISDN).
+
+Note: it is assumed that the UE IP address indicated by the AF is a public IP address. NAT considerations are left outside the scope of this document.
+
+#### AF obtaining the AF-specific UE External Identifier <a name="afobtaining"></a>
+
+3GPP has specified several mechanisms for allowing an AF to learn the AF-specific UE external identifier. These mechanisms include:
+
+1. The AF may invoke the Nnef\_UEId\_Get service operation for retrieving the AF‑specific UE external identifier. The procedure is specified in 3GPP TS 23.502 [11], clause 4.15.10 and allows the AF to use the UE IP or Ethernet address as an input to the query in order to obtain the AF-specific UE External Identifier of the UE.
```

Comment:
It would be useful to clearly state in which scenarios it would not be possible to make use of Nnef_UEId_Get

Reply:
I guess the only scenario where it would not be possible to use it is if the NEF does not implement this service or if it is disabled for whatever reason. But if the NEF implements the service and it is enabled, and if all the data is properly provisioned, then the AF could always use it.